.buildkite/pipeline.yaml

```diff
@@ -43,6 +43,16 @@ steps:
     <<: *defaults
     agent_query_rules: ["queue=build-unrestricted-large"]
 
+  - name: ":butterfly: changesets"
+    command: ".buildkite/scripts/changeset.sh"
+    timeout_in_minutes: 15
+    branches: "main"
+    if: build.author.name === "app/changeset-bot"
```

Comment:
The idea here is to take away the publishing step from the `changeset` GH action. This listens to whenever the changeset bot merges the version PR and launches the publish within buildkite instead.